src/System.Runtime.Serialization.Xml/tests/SerializationTestTypes/DataContractResolverLibrary.cs

```diff
@@ -168,4 +168,55 @@ public override Type ResolveName(string typeName, string typeNamespace, Type dec
             return t;
         }
     }
+
+    [Serializable]
+    public class SimpleResolver_Ser : DataContractResolver
+    {
+        public string defaultNS = "http://schemas.datacontract.org/2004/07/";
+        public override bool TryResolveType(Type dcType, Type declaredType, DataContractResolver KTResolver, out XmlDictionaryString typeName, out XmlDictionaryString typeNamespace)
+        {
+            string resolvedNamespace = string.Empty;
+            resolvedNamespace = defaultNS;
+            XmlDictionary dictionary = new XmlDictionary();
+            typeName = dictionary.Add(dcType.FullName);
+            typeNamespace = dictionary.Add(resolvedNamespace);
+            return true;
+        }
+
+        public override Type ResolveName(string typeName, string typeNamespace, Type declaredType, DataContractResolver KTResolver)
+        {
+            throw new NotImplementedException("Deserialization is supposed to be handled by the SimpleResolver_DeSer resolver.");
+        }
+    }
+
+    [Serializable]
+    public class SimpleResolver_DeSer : DataContractResolver
+    {
+        public string defaultNS = "http://schemas.datacontract.org/2004/07/";
```

Comment:
The field should be `private static readonly`. And the field name should be `s_defaultNs ` accroding to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md